tools/accuracy_checker/accuracy_checker/adapters/pose_estimation_hrnet.py

```diff
@@ -0,0 +1,133 @@
+"""
+Copyright (c) 2020-2021 Intel Corporation
```

Comment:
sorry for mess, but AC copyrights aligned with OV common copyrights rules (2018-2021)
```suggestion
Copyright (c) 2018-2021 Intel Corporation
```